pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -359,6 +359,16 @@ var _ = Describe("Generating fluentd config", func() {
 					alt_key kubernetes.event.metadata.uid
 					alt_tags "#{ENV['GENID_ALT_TAG'] || 'kubernetes.var.log.containers.logging-eventrouter-*.** kubernetes.var.log.containers.eventrouter-*.** kubernetes.var.log.containers.cluster-logging-eventrouter-*.** kubernetes.journal.container._default_.kubernetes-event'}"
 				</filter>
+				#flatten labels to prevent field explosion in ES
+				<filter ** >
+					@type record_transformer
+					enable_ruby true
+					<record>
+						kubernetes ${!record['kubernetes'].nil? ? record['kubernetes'].merge({"labels_alt": (record['kubernetes']['labels']||{}).map{|k,v| "#{k}=#{v}"}}) : {} }
```

Comment:
Does `labels_alt` reflect the final key name?

Reply:
I think it has to for atleast a release (or longer).  I defer to @lukas-vlcek but after and index is created we are unable to modify the field mapping.  Additionally, depending on time of upgrade we may have records with the map and others with the array so unless we intend to reindex and transform there is way to support both 'types'.  I'm open to alternate suggestions for naming